documentation/modules/auxiliary/server/socks4a.md

```diff
@@ -0,0 +1,60 @@
+
+## Verification Steps
+
+  1. Start msfconsole
+  2. Do: ```use auxiliary/server/socks4a```
+  3. Do: ```run```
+  4. Do: ```curl --proxy socks4a://localhost:1080 https://github.com```
+  5. You should see the source for the github homepage
+
+## Options
+
+  **SRVHOST**
+
+  The local IP address to bind the proxy to. The default value of `0.0.0.0` will expose the proxy to everything on the attackers network.
+
+  **SRVPORT**
+
+  The local port to bind the proxy to. The default value is `1080`, the standard port for a SOCKS4a proxy.
+
+## Scenarios
+
+  This module is great when pivoting across a network. Suppose we have two machines:
+
+  1. Attackers machine, on the `192.168.1.0/24` subnet.
+  2. Victim machine with two network interfaces, one attached to the `192.168.1.0/24` subnet and the other attached to the non-routable `10.0.0.0/24` subnet.
+
+  We'll begin by starting the socks4a proxy:
+  ```
+  msf > use auxiliary/server/socks4a
+  msf auxiliary(socks4a) > run
+  [*] Auxiliary module execution completed
+  [*] Starting the socks4a proxy server
+  msf auxiliary(socks4a) >
+  ```
+
+  Preparing to pivot across a network requires us to first establish a meterpreter session on the victim machine. From there, we can use the `autoroute` module to enable access to the non-routable subnet:
+
+  ```
+  meterpreter > run autoroute -s 10.0.0.0/24;
```

Comment:
I lifted this from some notes from a lab whereby I stacked up some commands for easier copy + paste, the semicolon survived, my bad.

I need to lean on your expertise on this one - is the module and the script one and the same, whereby the script applies sane defaults and generally makes it faster to execute? Is one preferred over the other?

We're in different timezones so I'll start by swapping it to say `script`, however I'm open to changing it to cover whatever the best practise is.